config/config.go

```diff
@@ -65,6 +67,7 @@ func FindConfigFile(start string) (string, error) {
 type configFile struct {
 	CreationRules    []creationRule    `yaml:"creation_rules"`
 	DestinationRules []destinationRule `yaml:"destination_rules"`
+	Path             string
```

Comment:
Sorry I missed this in the last review, but this struct is used to parse the .sops.yaml file, and I don't want this exposed in there. Can you move this out of the struct and just pass it in as a parameter to the functions?

Reply:
No problem. Tell me if it's ok now.